.github/workflows/tests.yml

```diff
@@ -65,7 +65,7 @@ jobs:
 
       - name: Check Go wrapper match
         run: |
-          if [ "$(git diff --ignore-space-at-eol wrappers/QuantumGravityBridge.sol/wrapper.go | wc -l)" -gt "0" ]; then
+          if [ "$(git diff --ignore-space-at-eol wrappers/QuantumGravityBridge.sol/wrapper.go wrappers/DAVerifier.sol/wrapper.go | wc -l)" -gt "0" ]; then
```

Comment:
Can this be done programmatically instead of hard-coding? E.g. by looping over `CONTRACTS`.

Reply:
Yeah sure, follow-up PR is fine